platform/vs/docker-syncd-vs/Dockerfile.j2

```diff
@@ -9,7 +9,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update
 
-RUN apt-get install -f -y iproute2 libcap2-bin
+RUN apt-get install -f -y libbsd0 libcap2-bin
```

Comment:
Is this build dependency? If yes, add some comment.
Alternatively, use `apt-get build-dep iproute2` ?

Reply:
I think `libbsd0` is a run-time dependency. And I want the original iproute2 from apt source is replaced by this PR. So I remove the original one.

Reply:
If it is a runtime dependency, is it better to install by `apt-get install -f` without specifying them one-by-one. In future, the dependency is free to change, and we are still good.